example/lib/snapshotter.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:mapbox_maps_flutter/mapbox_maps_flutter.dart';
+
+import 'page.dart';
+
+class SnapshotterPage extends ExamplePage {
+  SnapshotterPage()
+      : super(const Icon(Icons.camera_alt_outlined),
+            'Create a static map snapshot');
+
+  @override
+  Widget build(BuildContext context) {
+    return const SnapshotterPageBody();
+  }
+}
+
+class SnapshotterPageBody extends StatefulWidget {
+  const SnapshotterPageBody();
+
+  @override
+  State<StatefulWidget> createState() => SnapshotterPageBodyState();
+}
+
+class SnapshotterPageBodyState extends State<SnapshotterPageBody> {
+  SnapshotterPageBodyState();
+
+  GlobalKey _snapshotKey = GlobalKey();
+  MapboxMap? mapboxMap;
+  Image? snapshotImage;
+  Snapshotter? _snapshotter;
+  bool snapshotting = false;
+
+  @override
+  void dispose() {
+    _snapshotter?.dispose();
+    super.dispose();
+  }
+
+  _onMapCreated(MapboxMap mapboxMap) async {
+    this.mapboxMap = mapboxMap;
+
+    _snapshotter = await Snapshotter.create(
+      options: MapSnapshotOptions(
+          size: Size(width: 400, height: 400),
+          pixelRatio: MediaQuery.of(context).devicePixelRatio),
+    );
+    await _snapshotter?.style.setStyleURI(MapboxStyles.STANDARD);
```

Comment:
Perhaps a different style to show the difference: 

```suggestion
    await _snapshotter?.style.setStyleURI(MapboxStyles.OUTDOORS);
```